public/percent-complete/percent-complete.js

```diff
@@ -0,0 +1,265 @@
+import { JiraIssue } from "../shared/issue-data/issue-data.js";
```

Comment:
In `index.html.js`, I usually change:

```
import main from "./dist/main.js";
```

to

```
import main from "./main.js";
```

So I can load directly in the browser w/o source maps.  This import breaks that ability.